opentreemap/treemap/templates/treemap/partials/map_feature_popup.html

```diff
@@ -14,14 +14,14 @@
     </div>
     <div>
         <a href="{% url 'map_feature_detail' instance_url_name=request.instance.url_name feature_id=feature.pk %}"
-           class="btn btn-sm btn-secondary">{% trans "View All Details" %}</a>
+           class="btn btn-sm btn-secondary">{% trans "More Details" %}</a>
 
         {% if feature.is_plot or feature.is_editable %}
             <a disabled="disabled"
                data-always-enable="{{ last_effective_instance_user|map_feature_is_writable:feature }}"
                data-disabled-title="{% trans "Editing a tree's details is not available to all users" %}"
                data-href="{% url 'map_feature_detail_edit' instance_url_name=request.instance.url_name feature_id=feature.pk edit='edit' %}"
-               class="btn btn-sm btn-info">{% trans "Edit Details" %}</a>
+               class="btn btn-sm btn-info">{% trans "Edit" %}</a>
         {% endif %}
```

Comment:
Clicking on a bioswale still says "View All Details" and "Edit Details" in the popup for me.

The sidebar does say "More Details".